tools/travis-scripts/run-script.sh

```diff
@@ -75,8 +75,8 @@ function build_android()
     # popd
 
     # build cpp-tests
-    pushd $COCOS2DX_ROOT/tests/cpp-tests/proj.android
-    ./gradlew assembleRelease
```

Comment:
I think that we should use ./gradlew assembleRelease (at least for some configurations)